pkg/bgp/speaker/speaker.go

```diff
@@ -273,3 +352,25 @@ func convertEndpointSliceV1Beta1(in *slim_discover_v1beta1.EndpointSlice) *metal
 	}
 	return out
 }
+
+func nodeLabels(l map[string]string) *map[string]string {
+	n := make(map[string]string)
+	for k, v := range l {
+		n[k] = v
+	}
+	return &n
+}
+
+func podCIDRs(node *v1.Node) *[]string {
+	if node == nil {
+		return nil
+	}
+	podCIDRs := make([]string, 0, len(node.Spec.PodCIDRs))
+	if pc := node.Spec.PodCIDR; pc != "" {
```

Comment:
Comment here would be useful. For the uninitiated such as me this is a bit confusing :)

Reply:
add comment, lmk if this helps @joamaki I essentially link to the struct field comments which explain why those data access expressions look a bit magical.